src/main/java/org/opentripplanner/middleware/auth/Auth0UserProfile.java

```diff
@@ -2,21 +2,29 @@
 
 import com.auth0.jwt.interfaces.DecodedJWT;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.opentripplanner.middleware.models.AdminUser;
+import org.opentripplanner.middleware.models.ApiUser;
+import org.opentripplanner.middleware.models.AbstractUser;
+import org.opentripplanner.middleware.models.OtpUser;
+import org.opentripplanner.middleware.persistence.Persistence;
 
-import java.util.Arrays;
 import java.util.Date;
 
+import static com.mongodb.client.model.Filters.eq;
+
 /**
  * User profile that is attached to an HTTP request.
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Auth0UserProfile {
+    public final OtpUser otpUser;
+    public final ApiUser apiUser;
+    public final AdminUser adminUser;
     public String email;
```

Comment:
Why do we still have `email`, `email_verified`, `created_at` and `name`? They're never used.